### DISCUSSION TITLE
The Future of Zuma Blitz Remake

### DISCUSSION THREAD

**Discussion by exoduswalker** [ZumaBlitzRemake/ZumaBlitzRemake#23] [Announcements]
Hello, and sorry for the months of nothingness.

I would like to announce that Zuma Blitz Remake will be heading a different course now. Essentially, this project is going to be a **spiritual successor** instead of a full remake of Zuma Blitz Kroakatoa, for multiple reasons.

## Why?
- Recreating/scraping all of the lost graphics from videos/screenshots simply won't do because of the noticeable quality. Scroll down to "The Asset Problem" for further info.
- Some maps have been watermarked. These will be harder to remove, and quite frankly we might get in hot water with the artists that made Kroakatoa assets.
- Speaking of hot water, the nature of this project might be hit with a DMCA. A remake/revival project I can think of the top of my head is Angry Birds Refresh, which got DMCA'd this March and only lasted for a month, *even* if no money was being made off the game.
- Personally I've felt lazier porting the assets (maps/paths included) than actually coding in the features for the game. This includes editing the backgrounds to remove any unwanted elements.

## How?
- The codebase will be based off the current commit. I don't want the efforts here (Brendan's included) to go to waste.
- Stage13-10 will provide most, if not all of the path ideas and graphics. @lottieratworld will also be introducing herself as a graphics designer!
These changes shall be done later sometime since we're caught up with our own projects and IRL.
- This repository will be kept public so jakubg1 can cherry-pick specific commits if he'd like. Later on it may be privatized, but not immediately when the shift starts.
- Anyone is free to continue working, or start working on their forks. I understand how we want the "real" thing back as much as we want, but it isn't quite feasible. Furthermore, looking at forks' code may bring up some ideas on how to programmatically implement certain features.

This change of scenery will more than likely allow us to host a potential server without the fear of the entire project going down with a DMCA takedown. While it may be likely that this project will be ignored and have the code be relatively safe, I still won't take any risks anymore. I even privatized my Twitter account in panic of Angry Birds Refresh shutting down.

## The Asset Problem
Ever noticed the Spirit Eagle's Hot Frog state looking a little...crusty? Blurry? The main menu looking like it's been sharpened by AI, or specific assets being blurrier than they should be? Or maybe that the Spirit Turtle's warm and hot states don't look exactly like what they do in-game. **This is a fatal flaw of this project, and quite frankly some of these sources are from artist portfolio pages.** I don't think anyone who worked on ZB would sit right with this project's nature.

**TL;DR:** New assets and mostly same code to prevent potential DMCA takedowns.

**Comment by Charlotte-br560** [comment]
Understood, thanks for the update. Transitioning to a spiritual successor sounds like a wise move given the challenges with assets and potential legal issues. Excited to see the direction this project takes with Stage13-10's contributions and @lottieratworld's designs. Keeping the repository public for now is a good call

**Comment by jakubg1** [comment]
We would like to announce that the spiritual successor project, codenamed **Cosmic Crash** has been officially abandoned and it will never be continued. All existing work here will remain for the forseeable future, but officially there will be neither a Zuma Blitz Remake nor a spiritual successor finished.

If you looked forward to this project, ~~sucks to be you~~ you can always continue the project yourself! There is a bit of work still being done by @bchantech , so make sure to contact him (and also it's welcome to contact us as well) before proceeding. We're happy to help with any project-related questions.
There is also a private repository which contains some backends, alongside with replay checking. However, this is still maintained by @bchantech, so we aren't going to release it just yet.

We're sorry for the hopes we've made, it's just not happening this time!

**Comment by exoduswalker** [comment]
I will admit, it was mostly on my end. However, my negligence was not only the cause.

OpenSMCE in itself still needs work to do, and the theming was (if I recall correctly) not decided on - even the name had a planned changed, but that didn't happen either.

Simply put, it was easier to work with pre-made assets, and the work was only recreating the missing ones and coding in gameplay. Making a true spiritual successor is different as new assets needed to be made. However, the caveat was we had to essentially build from rubble. Not to mention, a "hype phase" often gets development up to speed.

In other words, I took from screenshots, videos, and contacts to try and recreate the assets for Kroakatoa Island. While this was not a problem for assets in the SWF file or assets in help pages, this is a major problem for assets or mechanics that were very unpopular, such as the Spirit Weasel and other, unpopular food items and screens.

In the meantime (or foreseeable future, rather), go play Zuma Blitz on Flashpoint. It's not the Kroakatoa version (rather the March 2011 beta), but it's still Zuma Blitz, right?
